mkdocs.yaml

```diff
@@ -1,9 +1,9 @@
-site_name: PyGerber Documentation
+site_name: PEC Documentation
 theme:
   language: en
 
-  logo: images/pygerber_white.png
-  favicon: images/pygerber.png
+  logo: 
```

Comment:
Have you checked if leaving those empty doesn't invalidate this file?

Reply:
As far as I know `logo` and `favicon` parameter is optional, but if we want to have these parameters filled, we need to decide what logo and favicon we want to have and then I will add these images to `docs/img` directory.